src/list.scss

```diff
@@ -909,4 +909,20 @@ $semantic-color: (
       @include navigation-byline-borderless();
     }
   }
+
+  &--interactive {
+    > * {
```

Comment:
This shouldn't be the universal selector. You should use `fd-list__item`. If a list item is interactive you need to have styling for all interactive states - hover, selected, selected+hover, active and the cursor should be a pointer.

Reply:
To add more to my comment: the interaction states should be applied to `fd-list--dropdown`